### DISCUSSION TITLE
Troubleshooting of incorrect entity value report

### DISCUSSION THREAD

**Discussion by Largelos** [WillCodeForCats/solaredge-modbus-multi#348] [Ideas]
Hi,
Recently, I faced with a strange behavior: all energy entities (export/import, per phase and summary) for Meter device (MTR-240-3PC1-D-A-MW, FW 1.0.32) increased by ~3 times. 
![image](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/80133928/ed1de0bc-5eb6-4991-8c20-036eb8d07e30)
How can I understand if this is an incorrect data output by the inverter / meter in modbus over tcp, or this is misinterpretation of the data by integration?
Is it possible to add raw registry data in debug log, before parsed hex values per device, which could be provided to support as proof of the firmware error?
Has anyone else encountered this behavior, why this happen and how to fix this?

**Comment by WillCodeForCats** [comment]
Hex values in debug or diagnostic are the raw values.

If you want raw modbus data you can turn debug on for pymodbus.

**Comment by Largelos** [comment]
Ok, thx.
In fact, data reported by meter really changed, even on meter's display. Surprising "reliability".
